docs: improve quickstart rover dev command text (#224)
apollographql/apollo-mcp-server: docs/source/quickstart.mdx

## Step 2: Run the MCP Server

1. From the root directory of your local repo, run `rover dev` to start a local graph with an MCP Server:
    ```sh showLineNumbers=false
    rover dev --supergraph-config ./graphql/TheSpaceDevs/supergraph.yaml \
    --mcp ./graphql/TheSpaceDevs/config.yaml
    ```

    This command:
    - Starts a local graph using the supergraph configuration
    - Starts an MCP Server with the `--mcp` flag
    - Provides a configuration file with MCP Server options

>     The [command reference](/apollo-mcp-server/command-reference#config-options) has a breakdown of available configuration options. 

Reviewer: ❤️  joining the dots

Reviewer 2: ```suggestion
    See the [command reference](/apollo-mcp-server/command-reference#config-options) for a list of available configuration options. 
```
Taking the Librarian suggestion for style consistency!
